### PR TITLE
Add webhook for CNV SSP resources

### PR DESCRIPTION
**Pull request by rajivnathan**
Prevents creation/updates of SSP resources

e2e tests: https://github.com/codeready-toolchain/toolchain-e2e/pull/887

### CODE REVIEW

File: pkg/webhook/validatingwebhook/validate_ssp_request_test.go [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+	"github.com/codeready-toolchain/member-operator/pkg/webhook/validatingwebhook/test"
+
+	userv1 "github.com/openshift/api/user/v1"
+	"github.com/stretchr/testify/require"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes/scheme"
+	"sigs.k8s.io/controller-runtime/pkg/client/fake"
+)
+
+func TestHandleValidateSSPAdmissionRequest(t *testing.T) {
```
Comment (alexeykazakov) [inline pkg/webhook/validatingwebhook/validate_ssp_request_test.go:21]: Let's add a test for the case when the request is allowed.

**Review by alexeykazakov — APPROVED** [review]
Looks good overall. But let's add more tests.

File: pkg/webhook/validatingwebhook/validate_ssp_request_test.go (hunk 1 above)
Comment (rajivnathan) [reply pkg/webhook/validatingwebhook/validate_ssp_request_test.go]: Added more tests in 40506aa4d1364d456cb1f690d9f5a86bc97ab362

**Review by alexeykazakov — APPROVED** [review]
Thanks!

**Pull request merged**